lib/manageiq/messaging/kafka/topic.rb

```diff
@@ -12,6 +12,11 @@ def publish_topic_impl(options)
           raw_publish(true, *topic_for_publish(options))
         end
 
+        def publish_topic_multi_impl(messages)
+          handles = messages.map { |message| raw_publish(false, *topic_for_publish(message)) }
```

Comment:
I don't know what this first param means, so I'm assuming it means "not-multi"?

Reply:
The first param means "wait", so for multi you don't want to wait on each message.  Honestly I think we can rip this out of raw_publish and let the _impl methods do the right thing depending on how they're calling it.